### ISSUE TITLE
sdwan_ipv4_acl_policy_definition resource can have ICMP message type, but not supported

### ISSUE THREAD

**Issue by cmohorea** [CiscoDevNet/terraform-provider-sdwan#314]
When configuring matching in vManage, when you specify protocol 1, it allows to specify ICMP type.
![image](https://github.com/user-attachments/assets/cda24b99-424e-4b5c-801f-f77e76c05e17)

in a schema, it seems to be a separate "icmpMessage" type, at least this is how it is understood by the provider when doing `terraform import`.
However, both type and sub-type values ("echo", "echo-reply", etc.) are not supported
Having this config (built based upon the imported data) results in error
```
      match_entries = [
        {
          source_data_ipv4_prefix_list_id = "50dc30b2-5915-4608-a302-0b8165ef603c",
          type = "sourceDataPrefixList",
        },
        {
          destination_data_ipv4_prefix_list_id = "0060c3a4-f9f2-4f15-b28e-ba60502d38eb",
          type = "destinationDataPrefixList",
        },
        {
          protocol = "1",
          type = "protocol",
        },
        {
          type = "icmpMessage",
        }
      ],
      name = "Access Control List",
    },
```
error:
```
Error: Invalid Attribute Value Match

  with sdwan_ipv4_acl_policy_definition.cn_qos_acl_marking,
  on sdwan-tf-import-main.tf line 12489, in resource "sdwan_ipv4_acl_policy_definition" "cn_qos_acl_marking":
12489: resource "sdwan_ipv4_acl_policy_definition" "cn_qos_acl_marking" {

Attribute sequences[2].match_entries[3].type value must be one of: ["dscp"
"sourceIp" "destinationIp" "class" "packetLength" "plp" "sourcePort"
"destinationPort" "sourceDataPrefixList" "destinationDataPrefixList"
"protocol" "tcp"], got: "icmpMessage"
```

**Comment by cmohorea** [comment]
same for the `sdwan_traffic_data_policy_definition`

**Comment by seconroy** [comment]
Released in v0.5.0.
